src/components/filters/filter-dynamic/filter-dynamic.component.ts

```diff
@@ -39,14 +40,16 @@ export class FilterDynamicComponent extends FilterBaseComponent {
         if (this.options && this.options.maxIndividualItems && this.options.maxIndividualItems + 1 >= this.filters.length ) {
             this.showTypeahead = false;
         }
-
     }
 
     selectOption(typeaheadOption: TypeaheadMatch) { 
         this.removeFilter();
         let idx = this.filters.findIndex(filter => filter.name === typeaheadOption.value);
         this.selected = this.filters[idx];
         this.addFilter(this.selected);
+        if (this.inputBox) {
+            this.inputBox.nativeElement.value = '';
```

Comment:
We should just be able to set `this.searchQuery = ''` here and the other places, however just for future reference, in the rare case where you do need to set the value of an element programmatically we should use the Renderer2 service which has a `setValue` method to do this. It also has many other similar methods which we should always use rather than directly manipulating an element. ([https://angular.io/api/core/Renderer2](https://angular.io/api/core/Renderer2))